domains/ariffhazmi.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Ariffhazmi's Website",
+  "repo": "https://github.com/Areplesapelagi/ariffhazmi.git",
+  "owner": {
+    "username": "Areplesapelagi",
+    "email": "muhdariffhazmi@gmail.com",

```

```suggestion
    "email": "muhdariffhazmi@gmail.com"

```
Trailing commas are not allowed